image/src/app.js

```diff
@@ -1,32 +1,41 @@
 const mqtt = require('mqtt')
 const { broker, port, topic, protocol } = require('./config/broker')
 const { urls } = require('./config/nextContainer.js')
-const { module_name, module_type } = require('./config/thisContainer.js')
 const fetch = require('node-fetch')
 
-const client = mqtt.connect(broker, {
-  port,
-  protocol,
-})
+if(broker.includes('tcp') || broker.includes('tls') || broker.includes('ws') || broker.includes('wss') || broker.includes('wxs') || broker.includes('alis')) {
```

Comment:
it's easier to whitelist the supported protocols, than try to blacklist all the unsupported ones. also using simple `includes` can heavily backfire: e.g. mqtt://browser.com contains "ws"

Reply:
That's right. I added a new approach